_posts/2020-12-07-Pivoting.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Metal3 Introduces Pivoting"
+date: 2020-12-07
+draft: false
+categories: ["metal3", "baremetal", "Pivoting", "Move"]
+author: Kashif Nizam Khan
+---
+
+`Metal3` project has introduced pivoting in its CI workflow. The motivation of
+pivoting is to move or pivot all the objects from the ephemeral/management
+cluster to target cluster. This blog post will briefly introduce the concept
+of pivoting and the impact it has on the overall CI workflow. For the rest of
+this blog, we refer ephemeral/management cluster as ephemeral cluster.
+
+## What is Pivoting?
+
+In the context of `Metal3` Provider, Pivoting is the process of moving
+`Cluster-API` and `Metal3` objects from the ephemeral k8s cluster to target
+cluster. In `Metal3`, this process is performed using the
+[clusterctl](https://cluster-api.sigs.k8s.io/clusterctl/overview.html) tool.
+provided by `Cluster-API`. clusterctl recognizes pivoting as `move`. During the
+pivot process clusterctl pauses any reconciliation on `Cluster-API` objects and
+this gets propagated to CAPM3 objects as well. Once all the objects are paused,
+the objects are created on the other side on target cluster and deleted on
+the ephemeral cluster.
+
+## Prerequisites
+Prior to the actual pivot process,target cluster should already have the
+provider components, ironic containers and CNI installed and running. To
+be exact the following points need to be addressed:
+
+* `clusterctl` is used to initialize both the bootstrap and target cluster.
```

Comment:
Maybe instead of bootstrap use either management or ephemeral, to not confuse readers ?